### PR TITLE
Fixes VSTS Bug 769480: Code fix preview window is too small

### PR DESCRIPTION
**Pull request by mkrueger**
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/769480

Resize was missing.

### CODE REVIEW

**Comment by mkrueger** [comment]
@monojenkins backport release-7.8

File: main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring/RefactoringPreviewTooltipWindow.cs [hunk 1]
```diff
@@ -135,6 +135,7 @@ void RequestPopup (Xwt.Rectangle rect)
 						if (temp.LineKind == LineKind.Normal && temp.TextMarkup == null) {
 							var newText = editor.GetMarkup (temp.Offset, temp.Length, new MarkupOptions (MarkupFormat.Pango, false));
 							diff.LineResults [i] = new LineResult (newText, temp.Offset, temp.Length, temp.LineKind, temp.XNeedsMeasure);
+							QueueResize ();
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring/RefactoringPreviewTooltipWindow.cs]: Why not resize at the end though?

Comment (mkrueger) [reply main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring/RefactoringPreviewTooltipWindow.cs]: Doesn't matter much hof often resize is queued - it's only resizing once.

Comment (Therzok) [reply main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring/RefactoringPreviewTooltipWindow.cs]: This seems like the kind of code that would end up called a lot from the UI thread. Regardless, looks okay to me.

**Pull request merged**